esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -147,12 +149,19 @@ class WsWuInfo
         const char* sourceAlias, MemoryBuffer &mb, bool forDownload);
     void copyContentFromRemoteFile(const char* sourceFileName, const char* sourceIPAddress,
         const char* sourceAlias, const char *outFileName);
+    void readDebugImportedStr()
+    {
+        Owned<IStringIterator> debugValues(&cw->getDebugValues("imported"));
+        if (debugValues->first())
+            debugValues->str(debugImportedStr);
```

Comment:
I think instead do:
```
   SCMStringBuffer imported;
   cw->getDebugValue(imported);
   logsPerWorkUnit = !isEmptyString(imported.str());
```

Reply:
Not sure it should be called 'logsPerWorkUnit' because the debugImportedStr is also used for # of thor slaves and log date.

Reply:
Where?
As far as I can see, it is only tested to see if set (not it's content), e.g.:
```
if (debugImportedStr.length()) //WU imported from a ZAP report
```

So is essentially a simple flag

Reply:
It is a simple flag. How bout changing the debugImportedStr to hasDebugImportedStr? The flag is not only used for getting logs, also used for finding numberOfSlaves for standard WU and for Imported WU.